tests/CMakeLists.txt

```diff
@@ -141,6 +187,18 @@ afr_module_dependencies(
         AFR::wifi
 )
 
+# BLE test
+# Only build on boards that support BLE.
+if(${BLE_SUPPORTED})
```

Comment:
You don't need to guard this, just make it depends on `AFR::ble` and it will be automatically disabled when `AFR::ble` is not available.